spec.html

```diff
@@ -5651,7 +5651,7 @@ <h1>Abstract Equality Comparison</h1>
         1. If Type(_x_) is Object and Type(_y_) is either String, Number, BigInt, or Symbol, return the result of the comparison ? ToPrimitive(_x_) == _y_.
         1. If Type(_x_) is BigInt and Type(_y_) is Number, or if Type(_x_) is Number and Type(_y_) is BigInt, then
           1. If _x_ or _y_ are any of *NaN*, *+&infin;*, or *-&infin;*, return *false*.
-          1. If the mathematical value of _x_ is equal to the mathematical value of _y_, return *true*; otherwise return *false*.
+          1. If the mathematical value of _x_ is equivalent to the mathematical value of _y_, return *true*; otherwise return *false*.
```

Comment:
`=`, no?

```suggestion
          1. If the mathematical value of _x_ = the mathematical value of _y_, return *true*; otherwise return *false*.
```

Reply:
I think it is a bit odd using `=` while having `the mathematical value of...`. Maybe it would be better having `ℝ(_x_) = ℝ(_y_)`?

Reply:
Sure, that reads better.